AirportDepartures.playground/Contents.swift

```diff
@@ -29,9 +60,35 @@ import UIKit
 //: d. Make one of the flights have a `nil` terminal because it has not been decided yet.
 //:
 //: e. Stretch: Look at the API for [`DateComponents`](https://developer.apple.com/documentation/foundation/datecomponents?language=objc) for creating a specific time
-
-
-
+//Part 2
+//a
+var flightBoard = DepartureBoard(airport: "LaGuardia Airport")
+
+var flight1Route = AirportRoute(departure: "LGA", arrival: "MIA")
+var flight1 = Flight(airline: "Spirit", refCode: "BR 219", status: .scheduled, route: flight1Route, departureTime: Date(), terminal: "2")
+
+var flight2Route = AirportRoute(departure: "LGA", arrival: "SFO")
+var flight2 = Flight(airline: "British Airways", refCode: "TL 347", status: .scheduled, route: flight1Route, departureTime: Date(), terminal: "4")
+
```

Comment:
The instructions had a couple scenarios that weren't created, (flight cancelled with departure nil and nil terminal). I created those cases and everything appears to have been handled correctly though!